index.bs

```diff
@@ -511,6 +564,53 @@ The <dfn method for=FileSystemFileHandle>createWritable(|options|)</dfn> method,
 
 </div>
 
+### The {{FileSystemFileHandle/createSyncAccessHandle()}} method ### {#api-filesystemfilehandle-createsyncaccesshandle}
+
+<div class="note domintro">
+  : |handle| = await |fileHandle| . {{FileSystemFileHandle/createSyncAccessHandle()|createSyncAccessHandle}}({ {{FileSystemFileHandleCreateAccessHandleOptions/mode}}: "in-place" })
+  :: Returns a {{FileSystemSyncAccessHandle}} that can be used to read/write from/to the file.
+     Changes made through |handle| might be immediately reflected in the file represented by |fileHandle|.
+     To ensure the changes are reflected in this file, the handle must be flushed or closed.
+
+    Creating a {{FileSystemSyncAccessHandle}} [=file entry/lock/take|takes an exclusive lock=] on the
+    [=FileSystemHandle/entry=] associated with |fileHandle|. This prevents the creation of
+    further {{FileSystemSyncAccessHandle|FileSystemSyncAccessHandles}} or {{FileSystemWritableFileStream|FileSystemWritableFileStreams}}
+    for the entry, until the access handle is closed.
+
+    The returned {{FileSystemSyncAccessHandle}} offers synchronous {{FileSystemFileHandle/read()}} and
+    {{FileSystemFileHandle/write()}} methods. This allows for higher performance for critical methods on
+    contexts where asynchronous operations come with high overhead i.e., WebAssembly.
```

Comment:
```suggestion
    contexts where asynchronous operations come with high overhead, i.e., WebAssembly.
```

Are there other such contexts? In this case it would be "e.g.".